README.md

```diff
@@ -498,6 +499,7 @@ Brazilian Portugese version : [here](https://github.com/LewisVo/Awesome-Linux-So
 - [GIMP](https://www.gimp.org/downloads/) - GIMP is a freely distributed program for such tasks as photo retouching, image composition and image authoring[![Open-Source Software][OSS Icon]](https://github.com/GNOME/gimp)
 - [Hugin](http://hugin.sourceforge.net/) - an easy to use cross-platform panoramic imaging toolchain based on Panorama Tools.
 - [ImageMagik](http://www.imagemagick.org/script/index.php) - ImageMagick is a suite of command-line utilities for modifying and working with images.
+- [imgp](https://github.com/jarun/imgp) - Blazing fast image resizer and rotator. [![Open-Source Software][OSS Icon]](https://github.com/jarun/imgp)
```

Comment:
You should add "command line" or "terminal" in the description.

Reply:
I will update the description.